python/pip_install/private/labels.bzl

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
```

Comment:
Please move this to somewhere under `python/private`. In the long term we would like to have all private files there. I'll create a small PR to do that in parallel, so that we could agree on the good location.